frontend/packages/operator-lifecycle-manager/src/components/install-plan.tsx

```diff
@@ -303,22 +302,23 @@ export const InstallPlanDetails: React.FC<InstallPlanDetailsProps> = ({ obj }) =
   return (
     <>
       {needsApproval && canPatchInstallPlans && (
-        <div className="co-m-pane__body">
-          <HintBlock title={t('olm~Review manual InstallPlan')}>
-            <p>
-              {t(
-                'olm~Inspect the requirements for the components specified in this InstallPlan before approving.',
-              )}
-            </p>
+        <Hint>
```

Comment:
Either need to keep `<div className="co-m-pane__body">` or add CSS so the spacing around the `<Hint>` is preserved.